### PR TITLE
Added docs for /cosmetics endpoint

### PR DESCRIPTION
**Pull request by MrAuro**
All information I found about it was by looking at this repo or by looking at how it is used at https://github.com/SevenTV/SevenTV 

Okayeg

### CODE REVIEW

**Comment by Excellify** [comment]
I believe the paints are going to have some changes. Drop_shadow will become drop_shadows and be able to include multiple. Some of the properties are optional, while some are not. 
[Here](https://github.com/SevenTV/SevenTV/blob/ca0a165b5b990932311e30011497a377bc5752ee/src/Sites/app/SiteApp.tsx#L170) is how the extension builds the style from the json. 
Also believe pr's should be to dev, not master.

image

**Comment by TroyDota** [comment]
@AnatoleAM will have to see if this is the correct design for the API documents.
This repo is planed to be deprecated within the next few months but adding the docs now sounds like a good idea.

**Review by AnatoleAM — CHANGES_REQUESTED** [review]
This is fine to document, however it's worth noting these two things are changing:

- The `animation` field will be dropped
- The `drop_shadow` field will be deprecated, in favor of `drop_shadows` (same structure, but as an array)

It may also be worth noting that
- The `color` field may be null
- Colors are all RGBA stored in a 32-bit integer and provide an example on how to decode them (i.e a javascript code sample)

File: docs/rest-api.md [hunk 1]
```diff
@@ -165,4 +165,128 @@ Get all active badges
...
+		}
+	],
+	"paints": [
+		{
+			"id": "61bede3db6b41ea54419bbb0",
+			"name": "Candy Cane",
+			"users": [
+				"24377667"
+            ],
+			"function": "linear-gradient",
```
Comment (Excellify) [inline docs/rest-api.md:220]: Might also be an idea to note the 3 accepted function: 

1. linear-gradient
2. radial-gradient
3. url

**Review by Excellify — APPROVED** [review]
Looks good to me

**Review by AnatoleAM — APPROVED** [review]

**Pull request merged**